### PR TITLE
add script to check z-scores of model versus artifact

### PR DESCRIPTION
**Pull request by KjellSwedin**

### CODE REVIEW

**Review by Ninicorn — APPROVED** [review]

File: tests/test_cgf.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+TIME_FIRST = 29
+TIME_SECOND =366-TIME_FIRST
+
+AGE_FIRST=(0.07671, 1.0)
+AGE_SECOND=(1.0, 5.0)
+
+ROUND_TO = 4
+
+def trans(inp, round_to):
+    return (inp - 10.0).round(round_to)
```
Comment (aflaxman) [inline tests/test_cgf.py:14]: Consider using `numpy.allclose` with `rtol=1e-4` as a cooler way to accomplish this https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Comment (aflaxman) [reply tests/test_cgf.py:14]: oops, i was reading too quickly; this is for pretty output, not for testing, right?  I get it now.  `f'{inp-10:.4f}'` will sometimes print prettier than rounding to 4 decimal places.

File: tests/test_cgf.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    for param in [(TIME_FIRST, AGE_FIRST), (TIME_SECOND, AGE_SECOND)]:
+        print(f'Days == {param[0]}')
+        sim.step(step_size=pd.Timedelta(days=param[0]))
+        #pop = sim.get_population().query('age >= {} and age <= {}'.format(param[1][0], param[1][1]))
+        pop = sim.get_population()
+        print(f'Pop len = {len(pop)}')
+        stunting_z = pipe_stunting(pop.index, skip_post_processor=True)
+        wasting_z = pipe_wasting(pop.index, skip_post_processor=True)
+        comp_data(art_stunting, art_wasting, stunting_z, wasting_z, param[1][0], param[1][1])
```
Comment (aflaxman) [inline tests/test_cgf.py:63]: I like tests that can pass or fail, so I recommend you throw in an `assert` here to automatically check that things are close enough.

**Review by aflaxman — APPROVED** [review]
Thanks Kjell!

File: tests/test_cgf.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    aw_mean = art_wasting.query('age_start >= {} and age_end <= {}'.format(age_start, age_end)).mean().mean()
+    mw_mean = model_wasting.mean()
+    aw_mean = trans(aw_mean, ROUND_TO)
+    mw_mean = trans(mw_mean, ROUND_TO)
+    print(f'Artifact wasting age mean = {aw_mean} versus Model age mean {mw_mean}')
+    aw_median = art_wasting.query('age_start >= {} and age_end <= {}'.format(age_start, age_end)).median().median()
+    mw_median = model_wasting.median()
+    aw_median = trans(aw_median, ROUND_TO)
+    mw_median = trans(mw_median, ROUND_TO)
+    print(f'Artifact wasting age mean = {aw_median} versus Model age mean {mw_median}')
```
Comment (aflaxman) [inline tests/test_cgf.py:46]: text in print out ("mean") does not match values (median)

File: tests/test_cgf.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    art_s_mean = art_stunting.query('age_start >= {} and age_end <= {}'.format(age_start, age_end)).mean().mean()
+    ms_mean = model_stunting.mean()
+    art_s_mean = trans(art_s_mean, ROUND_TO)
+    ms_mean = trans(ms_mean, ROUND_TO)
+    print(f'Artifact stunting age mean = {art_s_mean} versus Model age mean {ms_mean}')
+    art_s_median = art_stunting.query('age_start >= {} and age_end <= {}'.format(age_start, age_end)).median().median()
+    ms_median = model_stunting.median()
+    art_s_median = trans(art_s_median, ROUND_TO)
+    ms_median = trans(ms_median, ROUND_TO)
+    print(f'Artifact stunting age medial = {art_s_median} versus Model age median {ms_median}')
```
Comment (aflaxman) [inline tests/test_cgf.py:35]: typo s/medial/median/

**Pull request merged**